microsoft-edge/extensions-chromium/developer-guide/devtools-extension.md

```diff
@@ -1,55 +1,83 @@
+
+   The DevTools extension is loaded.
+
+1. In Microsoft Edge, open a new tab.
+
+1. Open DevTools (**F12**).
+
+1. Select the **Sample Panel** tool (tab).
+
+   The **Sample Panel** tool opens:
+
+   ![Microsoft Edge, showing DevTools on the side, with the extension's Sample Panel selected](./devtools-extension-images/solution-sample-panel.png)
 
-  ![The edge://extensions page in Microsoft Edge, showing the Developer mode and Load unpacked buttons](./devtools-extension-images/edge-extensions-page.png)
+1. Click the **Say Hello to the inspected page** button.
 
-Your DevTools extension should now be loaded. Open a new tab in Microsoft Edge, and then open DevTools by pressing **F12**.
+   An alert is displayed in the inspected page:
 
-The extension creates a **Sample Panel** tab in the DevTools window:
+   ![Microsoft Edge, showing the new extension panel in DevTools on the side, and an alert dialog window](./devtools-extension-images/devtools-extension-show-greeting.png)
 
-![Microsoft Edge, showing DevTools on the side, with the extension's Sample Panel selected](./devtools-extension-images/solution-sample-panel.png)
+1. Click the **OK** button to close the alert.
+
+1. Click anywhere in the inspected page.
+
+   A message is displayed in the extension panel: **You clicked on position (199, 509) in the inspected page**:
+
+   ![The sample panel in DevTools showing the clicked position message](./devtools-extension-images/devtools-extension-show-position.png)
+
+1. Select the **Console** tool.
+
+   A message appears in the **Console** tool: **Received response > _{xPosition: 199, yPosition: 509}_**:
+
+   ![The DevTools Console tool showing the clicked position message](./devtools-extension-images/devtools-extension-show-position-in-console.png)
+
+You downloaded, installed, ran, and tested the provided, finished DevTools extension sample.  In the next steps, you create the same sample by starting from an empty directory.  The following sections explain how the code works.
 
 
 <!-- ====================================================================== -->
-## Step 1 - Create a basic DevTools extension
+## Step 1: Create a basic Microsoft Edge extension
 
-A basic DevTools extension consists of two files, as shown in [Step 1 code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-extension/sample%201):
+If you want to create the files for each major step below, install a code editor such as [Visual Studio Code](https://code.visualstudio.com/), to follow the tutorial steps below to manually re-create the above sample DevTools extension.  You can read the code walkthrough below, presented as four major steps or phases.
 
-1. A manifest file.
+A basic extension for Microsoft Edge consists of two files, as shown in [Step 1 code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-extension/sample%201):
```

Comment:
That's not completely true. The devtools_page HTML file is not needed for a basic Microsoft Edge extension. It's only needed if you want to extend DevTools.

I understand, and agree with, the changes in this PR to make it clear that so-called "DevTools extensions" are really just Edge extensions.

So, perhaps change the h2 section header to "Step 1: Create a basic Microsoft Edge extension with a DevTools page".

And rephase this to:

```suggestion
A basic extension for Microsoft Edge consists of a manifest file. Because this extension extends DevTools, it also consists of a DevTools page. The two files are shown in [Step 1 code](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-extension/sample%201), and described below:
```